libraries/AC_WPNav/AC_WPNav.h

```diff
@@ -144,6 +144,9 @@ class AC_WPNav
     /// get_wp_destination waypoint using position vector (distance from home in cm)
     const Vector3f &get_wp_destination() const { return _destination; }
 
+    /// get origin using position vector (distance from home in cm)
+    const Vector3f &get_wp_origin() const { return _origin; }
```

Comment:
This is used for getting determining the heading to correct xtrack error during auto waypoints in Sub.